src/openems.yaml

```diff
@@ -0,0 +1,43 @@
+---
+repoUrl: 'https://github.com/OpenEMS/openems'
+name: OpenEMS
+description: >-
+  OpenEMS is a modular platform for energy management applications. It was
+  developed around the requirements of controlling, monitoring and integrating
+  energy storage systems together with renewable energy sources and
+  complementary devices and services.
+websiteUrl: 'https://openems.io/'
+license: EPL-2.0
+sdgs:
+  - 7
+programmingLanguages:
+  - Java
+  - HTML
+  - TypeScript
+  - Shell
+  - CSS
+  - JavaScript
+  - Dockerfile
+rating: 5
+ratingComment: ''
+contributionGuidelinesUrl: 'https://github.com/OpenEMS/openems#documentation'
```

Comment:
>  We are inviting third parties - like universities, hardware manufacturers, software companies, commercial and private owners,... - to use OpenEMS for their own projects and are glad to support them with their first steps

Are you sure this fits our user target?

Reply:
> This is not a contribution guide.

Well, it basically has two links for contribution guidelines, so I thought I'd link the part where both of them are mentioned.
I also thought about linking [the getting started part](https://openems.github.io/openems.io/openems/latest/gettingstarted.html), which is probably the best starting point when trying to contribute to the project.

> Are you sure this fits our user target?

Our target are programmers, who want to contribute to projects that support SDGs, right?
I admit it's not perfect for it, but I doubt many projects for SDG 7 will be – OSS-projects are hardly gonna be able to create powerlines in underdeveloped countries. OpenEMS can be used to optimize energy usage, though, and therefore might be useful for [7.3](https://sustainabledevelopment.un.org/sdg7#targets)?

I thought it's okay, but I can be convinced otherwise ^^

Reply:
Maybe this is a gray-zone case ^^. The link points to the documentation, but by contribution guide, I'm expecting a page explaining me how I can become useful to the project, what are they looking for, info on the roadmap etc.

I only briefly looked at it, so you'll be the better judge of it, but my initial impression was a "We built something. Feel free to use it, and we'll support you."; not a "We're building something and you can help us".
It does seem to fit the SDG 7, but I'm not sure on the contributor-friendly part of it :P

Reply:
> The link points to the documentation, but by contribution guide, I'm expecting a page explaining me how I can become useful to the project, what are they looking for, info on the roadmap etc.

True, but I haven't seen that for this project.
I'd probably even say that it doesn't have a contribution guideline (or at least I didn't find it).
Because of this, I thought I'd link the documentation on how to develope the project further, as that's the closest to it.

> I only briefly looked at it, so you'll be the better judge of it, but my initial impression was a "We built something. Feel free to use it, and we'll support you."; not a "We're building something and you can help us".

Hmmm. I'm gonna admit that I didn't spend too much time on it, since the website now makes it super easy to add a project in a short amount of time.
But I took it only as an additional offering. Like other OSS that also has a organisation behind it that offers support for its products.

Reply:
This feels like one of those projects where I wish we already had a metric where we can just mark a lack-luster contribution guide and its focus on using the project instead of developing it ^^'